wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -83,7 +83,7 @@ def main_nav(context):
     }
 
 
-@register.inclusion_tag("wagtailadmin/shared/breadcrumb.html", takes_context=True)
+@register.inclusion_tag("wagtailadmin/shared/breadcrumb_new.html", takes_context=True)
```

Comment:
drive-by suggestion: `breadcrumb-next.html` since we're working on the "next Wagtail editor interface"

Reply:
Sure, I can rename this! Great suggestion